resources/docker.core/windowsnano/docker-compose.yml

```diff
@@ -17,6 +17,7 @@ services:
     ports:
       - "${DOCKER_PLATFORM_PORT:-8090}:80"
     environment:
+      - VirtoCommerce:AutoInstallModuleBundles=false
```

Comment:
It won't work

Reply:
So only Platform 3 feature? Then i just del it.